### ISSUE TITLE
Traing loss

### ISSUE THREAD

**Issue by XyFighting** [NVlabs/MambaVision#48]
Hi, thanks for your amazing work!

I've encountered an issue with NaN losses when using MambaVision to train on CIFAR100 dataset. Could you suggest solutions to solve it?
![Training_loss](https://github.com/user-attachments/assets/7daf50d8-675c-4ee4-9910-5e7c5c0aaa25)

**Comment by ahatamiz** [comment]
Hi @XyFighting , the best recommendation is to lower the learning rate. The settings used in ImageNet experiment may not be optimal for other datasets.

**Comment by XyFighting** [comment]
Hi, thanks for your reply!
I have decreased the learning rate. However, when the training process proceeds after dozens of epochs, there are still NAN losses. Could you further suggest solutions to solve it?
